lib/src/config_handler.dart

```diff
@@ -109,6 +116,28 @@ class ConfigHandler {
     }
     configFile.writeAsStringSync(newTool, mode: FileMode.append);
   }
+
+  /// Disables the reporting capabilities if true is passed
```

Comment:
this seems like a confusing API, where true means disable?

Reply:
Thank you for pointing this out, this was a typo, if true is passed in, it will enable, just like the other method in `analytics.dart`